metsrw/utils.py

```diff
@@ -20,6 +35,13 @@ def lxmlns(arg):
 
 
 # CONSTANTS
-
 FILE_ID_PREFIX = 'file-'
 GROUP_ID_PREFIX = 'Group-'
+
+
+# URL helpers
+def urlencode(url):
```

Comment:
I think that for documentation purposes it would be good to have here a docstring explaining why are we doing this. I've tried myself to understand it and it took me a while. Please correct me if I'm wrong but what I understood is that the URI is not really being encoded (it already was) but escaped because it's being used in a [`XLink:href`](https://www.w3.org/TR/2001/REC-xlink-20010627/#link-locators) which has a limited set of allowed characters (type [anyURI](https://www.w3.org/TR/xmlschema-2/#anyURI))?